packages/atomic/src/components/search/atomic-sort-expression/atomic-sort-expression.tsx

```diff
@@ -24,9 +24,23 @@ export class AtomicSortExpression {
    * * `relevancy`
    * * `date ascending`/`date descending`
    * * `qre`
-   * * `<FIELD> ascending`/`<FIELD> descending`, where you replace `<FIELD>` with the name of a sortable field in your index (e.g., `criteria="size ascending"`).
+   * * `<FIELD> ascending`/`<FIELD> descending`, where you replace `<FIELD>` with the name of a sortable field in your index (`criteria="size ascending"`).
    *
-   * You can specify multiple sort criteria to be used in the same request by separating them with a comma (e.g., `criteria="size ascending, date ascending"`).
+   * You can specify multiple sort criteria to be used in the same request by separating them with a comma (`criteria="size ascending, date ascending"`).
+   * 
+   * .Combining Sort Criteria
+   * [%collapsible]
```

Comment:
this is an ASCIIDOC-specific syntax, I'm pretty sure it won't work in Storybook 🤔
To play safe, we may want to remove it, what do you think?

Reply:
Correct, it won't work in Storybook